src/user-icon/user-icon.scss

```diff
@@ -116,3 +131,59 @@
     }
   }
 }
+
+@mixin iui-user-icon-list {
+  display: flex;
+
+  .iui-user-icon:not(:first-child) {
+    margin-left: -$iui-xs;
+
+    .iui-initials {
```

Comment:
Do we really need to override these values?

Reply:
This `box-shadow` override is only needed for the smallest size because the shadow was larger than the margin between circles.  So even when the animated user icons were hovered, they would still slightly cover each other.  Mayank left a screenshot in a previous comment that has since been resolved.